airflow/models.py

```diff
@@ -2864,6 +2865,34 @@ def normalize_schedule(self, dttm):
 
         return dttm
 
+    def get_run_dates(self, start_date, end_date=None):
```

Comment:
Curious why we need a new method for this? The BackfillJob has to already have this information in order to execute the backfill, right? Looks like we already have a list of `_dag_runs` and a counter of the number that are finished: https://github.com/lyft/incubator-airflow/blob/a52489de3f402ddf72cb6f20d6b59965bc2d668b/airflow/jobs.py#L2032-L2039

Reply:
You are right. I think we don't need this method anymore.